### PR TITLE
Fix description and MD5 hash.

### PR DESCRIPTION
**Pull request by awholegunch**

### CODE REVIEW

**Comment by CuriousVini** [comment]
lgtm

File: cdap-common/src/main/resources/cdap-default.xml [hunk 1]
```diff
@@ -1172,8 +1172,11 @@
     <name>log.publish.partition.key</name>
     <value>program</value>
     <description>
-      Publish logs from a program or an app to same partition. Valid values are "application" and "program".
-      Changes to this property requires restart of all cdap applications.
+      Publish logs from a program or an application to the same partition, depending on
+      the key value. Valid values are "application" or "program". If set to "application",
```
Comment (chtyim) [inline cdap-common/src/main/resources/cdap-default.xml]: What do mean by "depending on the key value"? Maybe we don't need to say that? 
`Publish logs from a program or an application to the same partition. Valid values are "application" or "program".`

File: cdap-common/src/main/resources/cdap-default.xml [hunk 2]
```diff
@@ -1172,8 +1172,11 @@
...
+      logs from all the programs of an application go to the same partition. If set to
+      "program", logs from each program go to separate partitions. Changes to this
```
Comment (chtyim) [inline cdap-common/src/main/resources/cdap-default.xml]: I think it's better `logs from the same program go to the same partition`.

Comment (awholegunch) [reply cdap-common/src/main/resources/cdap-default.xml]: I think so to.

File: cdap-common/src/main/resources/cdap-default.xml (hunk 1 above)
Comment (awholegunch) [reply cdap-common/src/main/resources/cdap-default.xml]: Sure. And I will flip the order so it's always application and the program.

**Review by wolf31o2 — APPROVED** [review]
This LGTM... can we merge this to unblock docs builds?

**Comment by sreevatsanraman** [comment]
LGTM

**Pull request merged**